measurements/api/__init__.py

```diff
@@ -1,3 +1,5 @@
-from .docs import api_docs_blueprint
 from .private import api_private_blueprint
-from .measurements import api_blueprint
+
+from .measurements import get_version
+from .measurements import list_files
+from .measurements import list_measurements, get_measurement
```

Comment:
My understanding is that `api_blueprint` was a global object containing the routes. Since now we use another approach in which routing is implemented using a `YAML` file, we get directly the names of the APIs rather than getting them through the blueprint wrapper.

Also, these names are imported here &mdash; and not used &mdash; because here you're following the convention of making symbols available through `__init__.py` as they were part of `../api.py` but the implementation is in other files.

Reply:
Correct.